src/com/company/Threads.java

```diff
@@ -0,0 +1,21 @@
+package com.company;
+
+import org.apache.log4j.Logger;
+
+public class Threads extends Thread {
+    public final Counter counter;
```

Comment:
```suggestion
    private final Counter counter;
```
we don't need it to be public